CameraXBasic/gradle.properties

```diff
@@ -13,4 +13,4 @@ android.enableUnitTestBinaryResources=true
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
+# org.gradle.parallel=true
```

Comment:
Please undo changes to this file

Reply:
I don't know why git is pointing it as, there is no change in both lines